### DISCUSSION TITLE
[Feature request]: Create UX / UI Concept for users to see the assets assigned to them

### DISCUSSION THREAD

**Discussion by carlosvirreira** [Shelf-nu/shelf.nu#422] [Ideas]
### Is your feature request related to a problem? Please describe?

  As an IT administrator, I check out items to my colleagues. They typically sign off that they have access to the equipment. 
  
  I want them to be able to go to a link and see the assets assigned to them, including their signature date and comments. 
  

### Describe the solution you'd like

Perhaps this can be send on demand, with a link with authentication. 

This could allow for solving the issue of not having to create tons of accounts for them.

**Comment by DonKoko** [comment]
I have transformed this to a discussion as its not really an actionable issue. Once we figure out the UX and designs we can create an issue for it.
